Mark SL as unsupported (#1550)
theforeman/theforeman.org: _includes/manuals/2.0/3.1.1_supported_platforms.md

    * check the above repositories because the command can silently fail when subscription does not provide it: `yum repolist`
  * Apply all SELinux-related errata.
* CentOS, Scientific Linux or Oracle Linux 7
  * **Note:** The CentOS packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
  * Architectures: x86_64 only
  * [EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_can_I_install_the_packages_from_the_EPEL_software_repository.3F) is required
* Ubuntu 18.04 (Bionic)
  * Architectures: i386, amd64, aarch64
* Debian 10 (Buster)
  * Architectures: i386, amd64, aarch64

> PostgreSQL version 10 or newer. For EL 7 this is available from Software Collections.

Reviewer: Are we testing on PostgreSQL 10? I Think we were going with 12+

Reviewer 2: https://github.com/theforeman/theforeman.org/pull/1550#discussion_r404848830

    * check the above repositories because the command can silently fail when subscription does not provide it: `yum repolist`
  * Apply all SELinux-related errata.
* CentOS, Scientific Linux or Oracle Linux 7

>   * **Note:** The CentOS packages are not tested on Scientific Linux or Oracle Linux anymore. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.

Reviewer 2: ```suggestion
  * **Note:** The CentOS packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
```

  * Architectures: x86_64 only
  * [EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_can_I_install_the_packages_from_the_EPEL_software_repository.3F) is required
* Ubuntu 18.04 (Bionic)
  * Architectures: i386, amd64, aarch64
* Debian 10 (Buster)
  * Architectures: i386, amd64, aarch64

> PostgreSQL from Software Collections:

Reviewer 3: there is no SCL for debian/ubuntu

> * PostgreSQL version 10

Reviewer 3: This should be 12 i believe (even though 10 should work)

Reviewer 2: Debian Stretch ships version 11 and Ubuntu Bionic ships version 10.

Author: So should I mention it separately for Centos, Debian and Ubuntu ? Which is supported and tested pgsql 10 / 12 ?

Reviewer 2: No, PostgreSQL is PostgreSQL. We don't make a distinction between the OS it runs on. It just needs to be 10+. This also relates to a remote DB which may be on a different OS.

> * PostgreSQL version 12

Reviewer 2: ```suggestion
* PostgreSQL version 10
```

* Debian 10 (Buster)
  * Architectures: i386, amd64, aarch64

> PostgreSQL version 10 or newer
>   * For Ubuntu and Debian PostgreSQL is available from regular OS repositories.
>   * For CentOS PostgreSQL is available from Software Collections.

Reviewer 2: The current text implies you must use them from the OS, but that's not really true. You can also use packages from https://www.postgresql.org/ or other. It really just needs to be 10+. It probably also works with the AWS managed version. I'd limit it to
```suggestion
PostgreSQL version 10 or newer. For EL 7 this is available from Software Collections.
```
